src/screencapturekit.cs

```diff
@@ -335,4 +453,108 @@ interface SCStreamOutput {
 		[Export ("stream:didOutputSampleBuffer:ofType:")]
 		void DidOutputSampleBuffer (SCStream stream, CMSampleBuffer sampleBuffer, SCStreamOutputType type);
 	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	interface SCContentSharingPickerConfiguration {
+		[Export ("allowedPickerModes", ArgumentSemantic.Assign)]
+		SCContentSharingPickerMode AllowedPickerModes { get; set; }
+
+		[Export ("excludedWindowIDs", ArgumentSemantic.Strong)]
+		NSNumber [] ExcludedWindowIds { get; set; }
+
+		[Export ("excludedBundleIDs", ArgumentSemantic.Strong)]
+		string [] ExcludedBundleIDs { get; set; }
+
+		[Export ("allowsChangingSelectedContent")]
+		bool AllowsChangingSelectedContent { get; set; }
+	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SCContentSharingPicker {
+		[Static]
+		[Export ("sharedPicker")]
+		SCContentSharingPicker SharedPicker { get; }
+
+		[Export ("defaultConfiguration", ArgumentSemantic.Copy)]
+		SCContentSharingPickerConfiguration DefaultConfiguration { get; set; }
+
+		[NullAllowed]
+		[BindAs (typeof (int))]
+		[Export ("maximumStreamCount", ArgumentSemantic.Strong)]
+		NSNumber MaximumStreamCount { get; set; }
+
+		[Export ("active")]
+		bool Active { [Bind ("isActive")] get; set; }
+
+		[Export ("addObserver:")]
+		void AddObserver (SCContentSharingPickerObserver observer);
+
+		[Export ("removeObserver:")]
+		void RemoveObserver (SCContentSharingPickerObserver observer);
```

Comment:
to fix error BI1106: bgen: The parameter observer in the method ScreenCaptureKit.SCContentSharingPicker.AddObserver exposes a model (ScreenCaptureKit.SCContentSharingPickerObserver). Please expose the corresponding protocol type instead (ScreenCaptureKit.ISCContentSharingPickerObserver).

```suggestion
		[Export ("addObserver:")]
		void AddObserver (ISCContentSharingPickerObserver observer);

		[Export ("removeObserver:")]
		void RemoveObserver (ISCContentSharingPickerObserver observer);
```